src/sentry/workflow_engine/conditions/dual_write.py

```diff
@@ -0,0 +1,54 @@
+from collections.abc import Callable
+from typing import Any
+
+from sentry.rules.conditions.every_event import EveryEventCondition
+from sentry.rules.conditions.reappeared_event import ReappearedEventCondition
+from sentry.rules.conditions.regression_event import RegressionEventCondition
+from sentry.utils.registry import Registry
+from sentry.workflow_engine.models.data_condition import Condition, DataCondition
+from sentry.workflow_engine.models.data_condition_group import DataConditionGroup
+
+data_condition_translator_registry = Registry[
+    Callable[[dict[str, Any], DataConditionGroup], DataCondition]
+]()
+
+
+def translate_to_data_condition(data: dict[str, Any], dcg: DataConditionGroup):
+    translator = data_condition_translator_registry.get(data["id"])
+    return translator(data, dcg)
+
+
+@data_condition_translator_registry.register(EveryEventCondition.id)
+def create_every_event_condition(data: dict[str, Any], dcg: DataConditionGroup) -> DataCondition:
+    return DataCondition.objects.create(
+        condition=Condition.TRUTH,
+        comparison=True,
+        condition_result=True,
+        condition_group=dcg,
+    )
+
+
+@data_condition_translator_registry.register(ReappearedEventCondition.id)
+def create_group_comparison_reappeared_condition(
+    data: dict[str, Any], dcg: DataConditionGroup
+) -> DataCondition:
+    return DataCondition.objects.create(
+        type=Condition.GROUP_EVENT_ATTR_COMPARISON,
+        condition="state.has_reappeared",
+        comparison=True,
+        condition_result=True,
+        condition_group=dcg,
+    )
+
+
+@data_condition_translator_registry.register(RegressionEventCondition.id)
+def create_group_comparison_regression_condition(
+    data: dict[str, Any], dcg: DataConditionGroup
+) -> DataCondition:
+    return DataCondition.objects.create(
+        type=Condition.GROUP_EVENT_ATTR_COMPARISON,
```

Comment:
my worry with using the same type for different conditions is that they'll need to be serialized differently, but i think we can figure it out by making an enum for the possible `condition` values here. we also have conditions you can no longer select in the UI but we'll still have to show because we don't delete them